src/styles/block/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(/images/star.svg);
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
```

Comment:
It's a good practice to combine properties `background-image`, `background-repeat`, and others related to the background into a shorthand `background` property to make the code cleaner. For example: `background: url(/images/star.svg) no-repeat;`